### ISSUE TITLE
Support SSH connection name

### ISSUE THREAD

**Issue by Domain** [kbjr/terminus-title-control#2]
I just want the title be the name of my connection setting.

**Comment by kbjr** [comment]
I'm assuming you mean when using the SSH plugin? That shouldn't be too hard to do. What would you expect the title to be if you're not in an SSH connection?

**Comment by Domain** [comment]
Yes, I means SSH plugin. It should be fallback to default title when not in an SSH connection.

**Comment by psybertool** [comment]
> I'm assuming you mean when using the SSH plugin? That shouldn't be too hard to do. What would you expect the title to be if you're not in an SSH connection?

That would be awesome!

**Comment by tavisdxh** [comment]
That would be great if this feature has been supported!
